src/components/AuthorCard/AuthorCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { Image, Pressable, Text, View } from 'react-native';
+
+import styles from './styles';
+import globalStyles from '../../styles/globalStyles';
+
+type AuthorCardProps = {
+  name: string;
+  pronouns: string;
+  image: string;
+  bio: string;
+  artist_statement: string;
+};
+function AuthorCard({
+  name,
+  pronouns,
+  image,
+  bio,
+  artist_statement,
+}: AuthorCardProps) {
+  return (
+    <View style={styles.authorCardContainer}>
```

Comment:
only the image, author name, and pronouns should be in this view. the bio and statement can come afterwards